server/src/main/java/org/jboss/as/server/mgmt/NativeManagementResourceDefinition.java

```diff
@@ -56,7 +56,6 @@
     public static final SimpleAttributeDefinition SOCKET_BINDING = new SimpleAttributeDefinitionBuilder(ModelDescriptionConstants.SOCKET_BINDING, ModelType.STRING, false)
             .setXmlName(Attribute.NATIVE.getLocalName())
             .setValidator(new StringLengthValidator(1, Integer.MAX_VALUE, true, false))
-            .setAlternatives(ModelDescriptionConstants.INTERFACE)
```

Comment:
we should probably remove all references to INTERFACE / PORT by this time,
It would also simplify code that handles is quite a bit.